### PR TITLE
fixe s meatman icon bgu=

### PR DESCRIPTION
**Pull request by SmoSmoSmoSmok**
## About The Pull Request

 fixes this 
awsad2

## Why It's Good For The Game

 fixes this
awsad2

## Changelog

:cl:
fix: es this 
awsad2

/:cl:

### CODE REVIEW

**Comment by OneAsianTortoise** [comment]
image

**Comment by JohnFulpWillard** [comment]
image

File: fulp_modules/features/species/beefmen/mob/beefman.dm [hunk 1]
```diff
@@ -386,6 +386,8 @@
 		span_notice("The meat sprouts digits and becomes [beefboy]'s new [new_bodypart.name]!"),
 		span_notice("The meat sprouts digits and becomes your new [new_bodypart.name]!"))
 	new_bodypart.attach_limb(beefboy)
+	new_bodypart.update_limb(is_creating = TRUE)
+	beefboy.update_body_parts()
```
Comment (JohnFulpWillard) [inline fulp_modules/features/species/beefmen/mob/beefman.dm:390]: Looking into it, ``attach_limb()`` calls ``new_limb_owner.update_body()`` which calls ``update_body_parts(is_creating)``.

is it update_limbs that's the problem here?

Comment (SmoSmoSmoSmok) [reply fulp_modules/features/species/beefmen/mob/beefman.dm:390]: i think so, i tried adding update_limb by itself but it wouldnt work

**Comment by OperativeLyn** [comment]
image

**Pull request merged**